src/handlers/sqs-indexer/handle-bridge-in-event.ts

```diff
@@ -0,0 +1,267 @@
+import {TransactionReceipt, TransactionResponse} from '@ethersproject/abstract-provider'
+import {getNetworkByHolographId} from '@holographxyz/networks'
+import {TransactionDescription} from '@ethersproject/abi'
+
+import SqsService from '../../services/sqs-service'
+import {BridgeDirection, ContractType, PayloadType, SqsMessageBody} from '../../types/sqs'
+import {BridgeInArgs, BridgeInErc20Args, decodeBridgeInErc20Args} from '../../utils/bridge'
+import {capitalize, networkToChainId, sha3, storageSlot, toAscii} from '../../utils/utils'
+import {NetworkMonitor} from '../../utils/network-monitor'
+import {
+  create2address,
+  decodeDeploymentConfig,
+  DeploymentConfig,
+  deploymentConfigHash,
+} from '../../utils/contract-deployment'
+import {hexZeroPad} from '@ethersproject/bytes'
+
+async function handleBridgeInEvent(
+  networkMonitor: NetworkMonitor,
+  transaction: TransactionResponse,
+  network: string,
+  tags: (string | number)[],
+): Promise<void> {
+  const receipt: TransactionReceipt | null = await networkMonitor.getTransactionReceipt({
```

Comment:
similarly to the other other PR. This function should basically just send a job to the queue. Whatever information the indexer has at this point should go into the job object and then sent to the queue.